src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -127,6 +132,15 @@ public class ParameterDefinition
     public DbType? DbType { get; set; }
     public bool HasConfigDefault { get; set; }
     public object? ConfigDefaultValue { get; set; }
+
+    public bool Equals(ParameterDefinition other)
```

Comment:
what are the equals overloads used for? can you add comments

Reply:
not required, removed